todoapp/app/src/test/java/com/example/android/architecture/blueprints/todoapp/tasks/TasksViewModelTest.kt

```diff
@@ -1,5 +1,5 @@
 /*
- * Copyright 2017, The Android Open Source Project
+ * Copyright 2016, The Android Open Source Project
```

Comment:
2016?

Reply:
Yep, it is the year it was created on. Apparently that's what you're supposed to do.